Dockerfile

```diff
@@ -73,6 +72,8 @@ RUN \
     libx11-xcb-dev \
     libxcb-dri3-dev \
     libxcb-present-dev \
+    libxext-dev \
```

Comment:
I changed these because I sorted all packages, if they need to be closer to libexpat1, let me know I revert this.

Reply:
That's fine, thanks